### ISSUE TITLE
Missing claim: 'iat'

### ISSUE THREAD

**Issue by vxavictor513** [louketo/louketo-proxy#670]
# Missing claim: 'iat'

## Summary

I'm using Keycloak Gatekeeper with [IdentityServer4](https://github.com/IdentityServer/IdentityServer4) as OIDC provider, and hit the error below.

```
unable to verify the id token	{"error": "oidc: JWT claims invalid: missing claim: 'iat'"}
```

**Comment by Adriien-M** [comment]
I have the same issue, the `iat` claim is issued only for `id_token`, for `access_token` the claim `auth_time` is used...
In the meanwhile, did you find a workaround?

**Comment by vxavictor513** [comment]
@Adriien-M: Not really, except a brutal way to suppress the `iat` error in Gatekeeper, which I'm unsure whether it violates OIDC specs... What about you?
